app/jobs/upvs/fetch_digital_services_and_forms_list_job.rb

```diff
@@ -0,0 +1,52 @@
+require 'csv'
+require 'harvester_utils/downloader'
+
+class Upvs::FetchDigitalServicesAndFormsListJob < ApplicationJob
```

Comment:
Nazov DigitalServicesAndForms sa mi moc nepozdava. Ono to je mapovanie formularov na sluzby. Mozno `Upvs::ServiceFormMapping` ? Trochu sa mi to bije s mapping resources co mame v DH, pripadne mozno aj `Upvs::ServiceForm` ? cc @pavolzbell

Reply:
@jsuchal aha, siel by som `Upvs::DigitalServiceToForm` alebo `Upvs::ServiceToForm`, urcite by som dal spojku `To` nech je jasne, ze ide o mapovanie, ale `Mapping` by som na koniec nedaval prave kvoli DH, nech sa to nemyli

Reply:
@pavolzbell `ServiceWithForm`? lebo to mapovanie moze byt vlastne aj naopak. Imho je dokonca M:N.

Reply:
@jsuchal service to form mi pride lepsie: "sluzba na formular" / "sluzba k formularu" ako s with. to M:N neviem, no asi to nema zmysel delit na 2 modely, ci?

Reply:
> @pavolzbell `ServiceWithForm`? lebo to mapovanie moze byt vlastne aj naopak. Imho je dokonca M:N.

to `And` som sa tam snazil dat prave kvoli tomu.
`ServiceWithForm` mi dava zmysel

Reply:
@smith558 dajme ServiceWithForm a bude. aj zapis `ServiceWithForm.where(...).exists?` potom dava zmysel. Za mna dobre. 

https://en.wiktionary.org/wiki/bikeshedding

Reply:
PR bude updatnuty okolo 00:00. Este mam par veci co musim dokoncit pre polnocou.